src/main/java/entities/post.java

```diff
@@ -0,0 +1,118 @@
+package entities;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * The Post class contains all the necessary information of a post: its ID, title,
+ * description,tags,collaborators, deadline, and users who favourited it.
+ */
+public class Post extends Postable{
+    public static int numPostsCreated = 0;
+    /**The id is equal to the number of posts created.*/
+    private int id;
+    private Timer timer;
```

Comment:
I think we agreed to ditch the Timer object, the deadline is going to be controlled by calls to a controller in the main loop.